README.md

```diff
@@ -1 +1,224 @@
-# Kenny-training-flutter-practice
\ No newline at end of file
+# Kenny-training-flutter-practice
+
+![screenshot](resources/images/github-banner.png)
+
+# Flutter Boilerplate
+
+Fork this project then start you project with a lot of stuck prepare
+
+Base project made with much: . Contains Bloc, Route, and much more!
+
+## Table of contents
+
+- [How to Use](#how-to-use)
+- [Code Conventions](#code-conventions)
+- [Dependencies](#dependencies)
+- [Code structure](#code-structure)
+
+## Prerequisites
+
+- Flutter: >=3.7.0 <4.0.0
+- [Firebase staging](https://console.firebase.google.com/u/0/project/flutter-app-ec8dd/settings/iam)
+- [Firebase prod](https://console.firebase.google.com/u/0/project/template-app-prod-3a034/settings/iam)
+
+## How to Use
+
+## Run this template
+
+1. Download or clone this repo by using the link below:
+
+   ```
+   https://github.com/GoldenOwlAsia/mobile-flutter-template.git
+   ```
+
+2. Go to project root and execute the following command in console to get the required dependencies:
+
+   ```
+   flutter pub get
+   ```
+
+3. Now run the generator
+
+   ```
+   flutter packages pub run build_runner build
+   ```
+
+4. To run staging project
+
+   ```
+   flutter run --flavor staging --target lib/main_staging.dart
+   ```
+
+5. To run production project
+
+   ```
+   flutter run --flavor production --target lib/main_production.dart
+   ```
+
+## Use this template to create your project
+
+1. Download or clone this repo by using the link below:
```

Comment:
Please read the `Use this template to create your project` section and run the corresponding command to Update ReadMe, PackageName (Android), and BundleId (iOS) for this project.